ui/ui-frontend-common/src/app/modules/components/autocomplete/vitamui-autocomplete-multi-select/vitamui-autocomplete-multi-select.component.ts

```diff
@@ -252,7 +252,7 @@ export class VitamUIAutocompleteMultiSelectComponent implements ControlValueAcce
   }
 
   writeValue(preselectedOptionKeys: string[]) {
-    this.preselectedOptionKeys = preselectedOptionKeys;
+    this.preselectedOptionKeys = preselectedOptionKeys !== null ? preselectedOptionKeys.sort() : null;
```

Comment:
On doit pouvoir écrire :
```suggestion
    this.preselectedOptionKeys = preselectedOptionKeys?.sort();
```